### PR TITLE
fixing loggin issues

### PR DESCRIPTION
**Pull request by imariel2d**
Login wasn't redirecting when the session was valid.

**Evidence**
https://www.loom.com/share/07238abf256e43f0b9a0e6f4dc5f4298

### CODE REVIEW

File: src/domains/login/login.container.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import { APP_PATH } from 'routes';
+import { LoginComponent } from './login.component';
+
+export const Login = () => {
+  const isValidSession = useSelector((state) =>
+    state.app.get('isValidSession'),
+  );
+
+  return isValidSession ? <Redirect to={APP_PATH} /> : <LoginComponent />;
```
Comment (raiot) [inline src/domains/login/login.container.js]: Question, can this logic check go into the router? Like check the state there and make the redirection there, 
i.e. 
`<Route exact path="/login">
  {isValidSession ? <Redirect to={`/${APP_PATH}`} /> : <Login />}
</Route>`

Comment (raiot) [reply src/domains/login/login.container.js]: Even better would be something like `isLoggedIn` and the dependencies for that would be a valid token + a valid session

**Comment by tonymtz** [comment]
What's the difference between `isValidSession` and `isAppLoaded`? It seems like they both are used for the same purpose.

File: src/domains/login/login.container.js (hunk 1 above)
Comment (arielplas) [reply src/domains/login/login.container.js]: Is there a benefit to doing that? 🤔 

IMO it seems hard to read xD

Comment (raiot) [reply src/domains/login/login.container.js]: The benefit is that when you look for instance for an issue with redirection, it makes more sense to look for it in the router than it makes to go look for it in a specific component, besides its the router responsibility to tell the app where to go and under what conditions

Comment (arielplas) [reply src/domains/login/login.container.js]: True, makes more sense

**Review by tonymtz — APPROVED** [review]

**Pull request merged**